### PR TITLE
Add rate limiter to ordered consumer verticle 

### PR DESCRIPTION
**Pull request by pierDipi**
Build on top of https://github.com/knative-sandbox/eventing-kafka-broker/pull/1425

Part of #2125

## Proposed Changes

- Add rate limiter to ordered consumer verticle

**Docs**

### CODE REVIEW

File: data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OrderedConsumerVerticle.java [hunk 1]
```diff
@@ -29,31 +37,38 @@
...
     super(initializer, topics);
 
-    this.recordDispatcherExecutors = new HashMap<>();
+    final var vReplicas = Math.max(1, egress.getVReplicas());
+    final var tokens = maxPollRecords * vReplicas;
+    this.bucket = new LocalBucketBuilder()
+      .addLimit(Bandwidth.classic(tokens, Refill.greedy(tokens, Duration.ofSeconds(1))))
+      .withSynchronizationStrategy(SynchronizationStrategy.SYNCHRONIZED)
+      .withMillisecondPrecision()
+      .build();
```
Comment (pierDipi) [inline data-plane/dispatcher/src/main/java/dev/knative/eventing/kafka/broker/dispatcher/impl/consumer/OrderedConsumerVerticle.java:67]: @aslom if we inject this duration here, we can mock it and use minutes for unit tests and have more reliable low scale tests.

**Review by steven0711dong — APPROVED** [review]
LGTM

**Pull request merged**